4-projects/modules/standard_projects/main.tf

```diff
@@ -64,6 +67,27 @@ module "prod_project" {
   }
 }
 
+module "dev_project" {
+  source                      = "terraform-google-modules/project-factory/google"
+  version                     = "~> 7.0"
```

Comment:
bump to 8.0

Reply:
+1 - if we can do this for all projects that would be great